src/app/api/webhooks/clerk/route.ts

```
@@ -0,0 +1,46 @@
+import { NextRequest } from "next/server";
+import { upsertUser } from "@/features/users/db";
+import { deleteUser } from "@/features/users/db";
+
+export async function POST(request: NextRequest) {
+    try {
+        const event = await verifyWebhook(request);
+
+        switch (event.type) {
+            case "user.created":
+                case "user.updated":
+                    const clerkData = event.data;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)